src/components/number-picker/number-picker.component.ts

```diff
@@ -44,7 +43,18 @@ export class NumberPickerComponent implements ControlValueAccessor, OnDestroy, O
     @Input('aria-labelledby') labelledBy: string;
 
     /** Define the precision of floating point values */
-    @Input() precision: number = DEFAULT_PRECISION;
+    private _precision: number = null;
+
+    @Input()
+    get precision(): number {
```

Comment:
Is this setter/getter needed? We want to avoid them where possible as they are much slower than a simple variable value. Normally when it comes to inputs we just set a default initial value, which will always be used unless they override it which would simplify this to just:

```typescript
@Input() precision: number = Number.MAX_SAFE_INTEGER.toString().length - 1;
```

Reply:
I used the getter setter here to default to the MAX_SAFE_INTEGER when the precision is unset. I'll remove it now if that is not a behaviour we want to follow.

Reply:
The current implementation has that behavior, FYI.